recipes/runpynb/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "runpynb" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d276beb7ecaf92d7bbf4335e071343d3546afb36b8764a169a62eea8d48bc3dd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - nbconvert
+    - nbformat
+    - jupyter
```

Comment:
This isn't listed upstream (and should be pulled in if necessary by the other dependencies). Is there a particular reason you included it?
```suggestion
```

Reply:
Hi @dopplershift, thanks for reviewing!

Reply:
I plan to start list the jupyter dependency upstream too.